libs/core/toolbar/toolbar.component.ts

```diff
@@ -178,6 +181,9 @@ export class ToolbarComponent implements AfterViewInit, AfterViewChecked, CssCla
     /** @hidden */
     overflownItems: ToolbarItem[] = [];
 
+    /** @hidden */
+    spacerUsed = false;
```

Comment:
if you use `contentChildren` signal, this can also be a computed signal and then its value will change when the spacers are updated.

Reply:
check out the changes I have pushed